compiler/src/test/scala/org/scalajs/core/compiler/test/InternalAnnotationsTest.scala

```diff
@@ -27,8 +27,8 @@ class InternalAnnotationsTest extends DirectTest with TestHelpers {
   }
 
   @Test
-  def sjsDefinedAnonymousClass: Unit = {
-    test("SJSDefinedAnonymousClass")
+  def anonymousJSClass: Unit = {
+    test("AnonymousJSClass", "scala.scalajs.js.annotation.internal.AnonymousJSClass")
   }
```

Comment:
There will be a silent conflict with https://github.com/scala-js/scala-js/pull/2991/commits/b3f5839e7b4f88e0c94457f0d37d7befdb392ab4 here. The fully qualified name won't be necessary.